src/Products/SiteAccess/tests/testVirtualHostMonster.py

```diff
@@ -198,8 +258,3 @@ def test_add_manage_addVirtualHostMonster(self):
         hook = queryBeforeTraverse(self.root, VirtualHostMonster.meta_type)
         self.assertTrue(hook)
 
-def test_suite():
```

Comment:
As said in the commit message, this is a small difference from the version on master, because in 2.13 branch, the `test_suite` function was stil here, so tests from the new added test class were not run.